src/test/java/uk/gov/companieshouse/company/profile/service/CompanyProfileServiceTest.java

```diff
@@ -1474,6 +1482,8 @@ void deletePscLinkDataAccessExceptionUpdate() throws ApiErrorResponseException {
     @Test
     @DisplayName("Put company profile")
     void putCompanyProfileSuccessfully() throws IOException {
+        when(companyProfileRepository.findById(MOCK_PARENT_COMPANY_NUMBER)).thenReturn(Optional.of(EXISTING_PARENT_COMPANY_PROFILE_DOCUMENT));
+        EXISTING_PARENT_COMPANY_PROFILE_DOCUMENT.getCompanyProfile().getLinks().setUkEstablishments(null);
```

Comment:
how come we need these lines in this test?

Reply:
If the company we are adding has a parent we will get that parent from the repository to set the uk-establishments link so we need to mock that behaviour fro the database